charts/team-ns/templates/tekton-tasks/git-clone.yaml

```diff
@@ -13,12 +15,16 @@ metadata:
 spec:
   stepTemplate:
     computeResources:
+      {{- if $custom }}
+        {{- $v.resources.gitcloneTask.resources | toYaml | nindent 6 }}
+      {{- else }}
       limits:
         cpu: "2"
         memory: 2Gi
       requests:
         cpu: 500m
         memory: 512Mi
```

Comment:
Do we maybe want to set these defaults lower? I think its pretty high for a simple git-clone task

Reply:
No because we want to provide a good initial user experience